database/pom.xml

```
@@ -85,11 +86,6 @@
         <artifactId>mssql-jdbc</artifactId>
         <version>${version.sqlserver}</version>
       </dependency>
```

```suggestion
      </dependency>
      <dependency>
        <groupId>com.ibm.db2</groupId>
        <artifactId>jcc</artifactId>
        <version>${version.db2}</version>
      </dependency>
      <dependency>
        <groupId>com.ibm.db2.jcc</groupId>
        <artifactId>db2jcc4</artifactId>
        <version>${version.db2}</version>
      </dependency>
```

Should we add these dependencies to the `dependencyManagement` section? It safes us to redeclare the version again at some places.